src/web/src/browser/views/components/HomePage.tsx

```diff
@@ -227,15 +227,15 @@ const WhySection = () => (
             Contact us
           </h2>
           <p>
-            Please give us your feedback via a{' '}
-            <a
-              className="text-underline text-primary"
-              href="https://github.com/GSA/fedramp-automation/issues"
-            >
-              Github issue
-            </a>
-            .
+            Your feedback is important to us. Please send us any problems or
+            feature requests to our team to help make ASAP better for everyone.
           </p>
+          <a
+            className="usa-button radius-pill bg-theme-deep-blue margin-top-1 text-uppercase"
+            href="https://github.com/GSA/fedramp-automation/issues/new/choose"
```

Comment:
Would it be possible to add a function named something like `getNewIssueUrl` to [github.ts](https://github.com/18F/fedramp-automation/blob/master/src/web/src/shared/domain/github.ts) to return this URL?

That would enable having it point to 18F for now, then switch to GSA when we handover.

The other Github URLs are passed into the UI here - https://github.com/18F/fedramp-automation/blob/master/src/web/src/browser/index.ts#L49-L53

Reply:
Thanks for your feedback @danielnaab; I've added the functions and the links still work.